cyclus/system.py

```diff
@@ -23,7 +23,7 @@ def curio():
 
 @lazyobject
 def asyncio():
```

Comment:
This should look like the curio replacement above, but replace curio with asyncio.